recipe/0005-fix-setup-gdal-libname.patch

```diff
@@ -0,0 +1,15 @@
+--- swig/python/setup.py.orig	2022-08-04 13:00:57.659818933 +1000
++++ swig/python/setup.py	2022-08-04 13:04:30.130263637 +1000
```

Comment:
Caution! swig/python/setup.py is only used by autoconf/nmake buids, and is going t be removed per https://github.com/OSGeo/gdal/pull/6110
With cmake, the source file is swig/python/setup.py.in, which is processed at cmake configuration time to $build_dir/swig/python/setup.py

Reply:
Thanks for clarifying @rouault that makes more sense. Is swig/python/setup.py.in only processed when `BUILD_PYTHON_BINDINGS=ON` is passed to `cmake`? 
Is it possible to just build the C++ library and utils and then separately build the python extension?

Reply:
> Is swig/python/setup.py.in only processed when `BUILD_PYTHON_BINDINGS=ON` is passed to `cmake`?

yes (which is the default if Python is detected)

> Is it possible to just build the C++ library and utils and then separately build the python extension?

semi sort-of. https://github.com/OSGeo/gdal/blob/a03e48274244105360c3af3477fa72738b50a3ec/.github/workflows/cmake_builds.yml#L215 shows a workflow where you build the C++ lib&utils, generate a python sdist, and then build it.